src/api/protectedApiClient.ts

```diff
@@ -23,8 +30,14 @@ const changePassword = (request: {
     .catch((e) => e);
 };
 
+const setContacts: (contactInfo: SetContactsRequest) => Promise<void> = (
+  contactInfo: SetContactsRequest,
+) =>
+  AppAxiosInstance.post(ProtectedApiClientRoutes.SET_CONTACTS, contactInfo, {});
```

Comment:
What's the 3rd parameter {} here? Options? Why do we have to pass an empty object?

Reply:
ngl I have absolutely no clue, in fact this really isnt relevant to this PR so ill get rid of it 🔪